src/test/scala/io/dockstore/release_1_9/Organizations.scala

```diff
@@ -1,15 +1,16 @@
 package io.dockstore.release_1_9
```

Comment:
Is `release_1_9` still appropriate? Is this a Scala version or a Dockstore version?

Reply:
It's the Dockstore version. There was a new set of tests written for 1.9 (orgs, new home pages), so they got put in there. Given that we're sort of starting from scratch again, I'll get rid of it.

098a684582f29d5989d9da812a4bd0378f748033